templates_apprunner/templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
+Parameters:
+  SysdigRoleName:
+    Type: String
+    Default: "SysdigAgentlessRole"
+    Description: Unique role for monitoring AWS accounts
+  SysdigExternalID:
+    Type: String
+    Description: Sysdig ExternalID required for the policy creation
+  SysdigTrustedIdentity:
+    Type: String
+    Description: Sysdig Trusted identity required for policy creation
+  SysdigEBName:
```

Comment:
I'm not sure if this role should be configurable. If we make it configurable, we will need an additional input to collect this role name in the sysdig UI.

Reply:
This is the `name` variable in terraform for EB. Its optional and  `default` value is set to `sysdig`. But customers can override this value